src/components/ItemTag.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <span class="item-tag">
+    <span :style="{ background: contributorTagColor(item.name) }"></span>
+    <span>{{ item.name }}</span>
+  </span>
+</template>
+
+<script>
+import { contributorTagColor } from '@/util/okr';
+
+export default {
+  name: 'ItemTag',
+
+  props: {
+    item: {
+      type: Object,
+      required: true,
+    },
+  },
+
+  methods: {
+    contributorTagColor,
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+.item-tag {
+  display: inline-flex;
+  gap: 0.25rem;
+  align-items: center;
+  white-space: nowrap;
+
+  span:first-of-type {
```

Comment:
Litt tydeligere intensjon om man klasser den i stedet kanskje?

Altså `.beskrivende-klasse` i stedet for `span:first-of-type`.